layers/stateless/sl_shader_object.cpp

```diff
@@ -0,0 +1,214 @@
+/* Copyright (c) 2015-2023 Nintendo
+ * Copyright (c) 2015-2023 LunarG, Inc.
```

Comment:
2015?

Also this will fail CI. See CONTRIBUTING.md

```suggestion
/* Copyright (c) 2023 Nintendo
 * Copyright (c) 2023 LunarG, Inc.
```